src/containers/flows/MintWheels/index.tsx

```diff
@@ -391,6 +375,18 @@ const MintWheelsFlowContainer = () => {
 			: getBannerButtonText(dropStage, canOpenWizard);
 	};
 
+	const bannerImage = () => {
+		return banner;
```

Comment:
We'd need to store the data and call it - but idea here would be pass ```IndustryType``` to ```bannerImage```, and return the relevant image url.